libsqlite3-sys/build.rs

```diff
@@ -127,6 +127,19 @@ mod build_bundled {
         }
         println!("cargo:rerun-if-env-changed=SQLITE_MAX_EXPR_DEPTH");
 
+        if let Ok(extras) = env::var("LIBSQLITE3_FLAGS") {
```

Comment:
Thinking more about this, we need to limit some flags perhaps. Certain runtime checks are disabled for bundled builds (since we know the specific configuration). I think we still check threadsafety, but am unsure. If we do not, we should at least issue an error here if they try to disable it at compile time.

Reply:
> Thinking more about this, we need to limit some flags perhaps. Certain runtime checks are disabled for bundled builds (since we know the specific configuration). I think we still check threadsafety, but am unsure. If we do not, we should at least issue an error here if they try to disable it at compile time.

Only `rusqlite` requires threadsafety. Crates which depend on `libsqlite3-sys` directly may not have such constraint.

Reply:
Oh, good point!